doc/source/myrocks/differences.rst

```diff
@@ -0,0 +1,29 @@
+.. _myrocks_differences:
+
+========================================================
+Differences between Percona MyRocks and Facebook MyRocks
+========================================================
+
+The original MyRocks was developed by Facebook
+and works with their implementation of MySQL.
+*Percona MyRocks* is an implementation for Percona Server
```

Comment:
It's possible to interpret this that "Percona MyRocks" is a parallel implementation to Facebook MyRocks. It is not, it's a port from Facebook, better call it like that

Reply:
👍